core/src/pages.rs

```diff
@@ -35,8 +35,8 @@ pub const WASM_PAGE_SIZE: usize = 0x10000;
 /// native page size, so can vary.
 pub const GEAR_PAGE_SIZE: usize = 0x4000;
 
-static_assertions::const_assert!(WASM_PAGE_SIZE < u32::MAX as usize);
-static_assertions::const_assert_eq!(WASM_PAGE_SIZE % GEAR_PAGE_SIZE, 0);
+const _: () = assert!(WASM_PAGE_SIZE < u32::MAX as usize);
+const _: () = assert!(WASM_PAGE_SIZE % GEAR_PAGE_SIZE == 0);
```

Comment:
Is it possible to use `assert_eq` here? As it shows left and right if not equal.

Reply:
not yet: rust-lang/rust#74925